gui/default/index.html

```diff
@@ -744,7 +744,9 @@ <h4 class="panel-title">
                     <span ng-switch-when="paused"><span class="hidden-xs" translate>Paused</span><span class="visible-xs" aria-label="{{'Paused' | translate}}"><i class="fas fa-fw fa-pause"></i></span></span>
                     <span ng-switch-when="unused-paused"><span class="hidden-xs" translate>Paused (Unused)</span><span class="visible-xs" aria-label="{{'Paused (Unused)' | translate}}"><i class="fas fa-fw fa-unlink"></i></span></span>
                     <span ng-switch-when="disconnected"><span class="hidden-xs" translate>Disconnected</span><span class="visible-xs" aria-label="{{'Disconnected' | translate}}"><i class="fas fa-fw fa-power-off"></i></span></span>
+                    <span ng-switch-when="disconnected-inactive"><span class="hidden-xs" translate>Disconnected (Inactive)</span><span class="visible-xs" aria-label="{{'Disconnected (Inactive)' | translate}}"><i class="fas fa-fw fa-power-off"></i></span></span>
                     <span ng-switch-when="unused-disconnected"><span class="hidden-xs" translate>Disconnected (Unused)</span><span class="visible-xs" aria-label="{{'Disconnected (Unused)' | translate}}"><i class="fas fa-fw fa-unlink"></i></span></span>
+                    <span ng-switch-when="unused-disconnected-inactive"><span class="hidden-xs" translate>Disconnected (Unused, Inactive)</span><span class="visible-xs" aria-label="{{'Disconnected (Unused, Inactive)' | translate}}"><i class="fas fa-fw fa-unlink"></i></span></span>
```

Comment:
This one I find a bit confusing. If we don't share anything with a device, does it matter whether the connection had been established short or long ago?

```suggestion
                    <span ng-switch-when="unused-disconnected-inactive"><span class="hidden-xs" translate>Disconnected (Unused)</span><span class="visible-xs" aria-label="{{'Disconnected (Unused)' | translate}}"><i class="fas fa-fw fa-unlink"></i></span></span>
```

Reply:
Done.

Reply:
Your latest commit removed the whole case though, so nothing will be displayed for `unused-disconnected-inactive`.  We should just display the same label as for `unused-disconnected`, since both are still true.

Reply:
Thanks for noticing! I've fixed this now by reverting to what the code looked originally. What do you think?

Reply:
I meant dropping the `, Inactive` part within parentheses. Just use the same label `Disconnect (Unused)` as stated above.

Reply:
Done. I've reverted the JS change and re-added the HTML. There's a lot of duplicated code in that whole HTML block, which I think could be simplified quite a bit, but I'll leave this for a different PR.